testsuite/tests/singlecluster/defaults/merge/test_gateway_default_merge_auth_policy.py

```diff
@@ -0,0 +1,73 @@
+"""Test gateway level default merging with and being partially overriden by another policy."""
+
+import pytest
+
+from testsuite.httpx.auth import HttpxOidcClientAuth
+from testsuite.kuadrant.policy.authorization.auth_policy import AuthPolicy
+from testsuite.kuadrant.policy.rate_limit import RateLimitPolicy, Limit, Strategy
+from testsuite.oidc import OIDCProvider
+from testsuite.oidc.keycloak import Keycloak
+
+pytestmark = [pytest.mark.kuadrant_only, pytest.mark.limitador]
```

Comment:
`pytest.mark.limitador` -> `pytest.mark.authorino`
Not only here but also in other modules testing AuthPolicy